lib/Service/ContactsIntegration.php

```diff
@@ -106,14 +106,12 @@ public function getMatchingRecipient(string $term): array {
 	 * @return false|null|string
 	 */
 	public function getPhoto(string $email) {
-		$result = $this->contactsManager->search($email, ['EMAIL']);
-		foreach ($result as $contact) {
-			if (!isset($contact['PHOTO']) || empty($contact['PHOTO'])) {
-				continue;
+		$result = $this->contactsManager->search($email, ['EMAIL'], ['escape_like_param' => false]);
```

Comment:
there is a better way: `'strict_search' => true,`

we don't need nor want wildcard matches for the email address

Reply:
yep, since  v 23.0 =)